src/integer.rs

```diff
@@ -42,11 +42,18 @@ impl FromStr for Integer {
     type Err = ParseIntError;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let (s, radix) = if s.starts_with("0x") {
+            let s = s.trim_start_matches("0x");
```

Comment:
This will want to be `trim_left_matches` as `str::trim_start_matches` is only available on Rust 1.30.0.

Reply:
Oh, my bad, force of `edition2018` habit, you know :)

The only thing where min rustc version can be found is a Travis configuration file, so this was not very obvious. shields.io provides the badge for minimal rustc version, btw:
```
![Minimum rustc version](https://img.shields.io/badge/rustc-1.26+-yellow.svg)
```